opencti-platform/opencti-graphql/src/modules/entitySetting/entitySetting.graphql

```diff
@@ -45,6 +54,7 @@ type EntitySetting implements InternalObject & BasicObject {
   availableSettings: [String!]! @auth
   created_at: DateTime! @auth
   updated_at: DateTime! @auth
+  overviewLayoutCustomization: [OverviewLayoutCustomization!] @auth
```

Comment:
Missing a ! after the []

Reply:
This must be nullable because the overview layout customization will be available on a subset of entity types.